application/statics/statics.go

```diff
@@ -4,16 +4,20 @@ import (
 	"archive/zip"
 	"bufio"
 	"crypto/sha256"
+	"debug/buildinfo"
 	_ "embed"
 	"encoding/json"
 	"fmt"
 	"io"
 	"io/fs"
 	"net/http"
+	"os"
 	"path/filepath"
 	"sort"
 	"strings"
+	"time"
 
+	embedfs "github.com/alimy/tryst/embed"
```

Comment:
Please inline this package. I don't see the need to import from upstream since it's only ~50 lines of wrapper code.

Reply:
can wrap this code yourself just reference [参考文章- 为 go:embed 嵌入的资源开启条件缓存](https://taoshu.in/go/embed-web-cache.html), and i will just close this issue because i had use  "github.com/alimy/tryst/embed" package in some other project by mit license.

Reply:
ok, i use other implement for same effect.